GETTING_STARTED.md

```diff
@@ -0,0 +1,240 @@
+# RancherOS Getting Started Guide
+
+RancherOS is a lightweight Linux distro designed from the ground up to run Docker containers. This guide will provide detailed instructions on how to get started using RancherOS, and will include:
```

Comment:
This guide *will* provide -> This guide provides
..and *will* include -> including: